WalletWasabi.Daemon/Config.cs

```diff
@@ -184,6 +190,8 @@ public Config(PersistentConfig persistentConfig, string[] cliArgs)
 
 	public bool EnableGpu => GetEffectiveValue<BoolValue, bool>(nameof(EnableGpu));
 	public string CoordinatorIdentifier => GetEffectiveValue<StringValue, string>(nameof(CoordinatorIdentifier));
+	public decimal MaxAllowedCoordinationFeeRate => GetEffectiveValue<DecimalValue, decimal>(nameof(MaxAllowedCoordinationFeeRate));
+	public FeeRate MaxAllowedCoinjoinMiningFeeRate => GetEffectiveValue<FeeRateValue, FeeRate>(nameof(MaxAllowedCoinjoinMiningFeeRate));
```

Comment:
I followed this, without implementing a Rate class.
I believe by using `decimal` instead of FeeRate the code is ok, and we can improve it later without issue.